content/docs/reference/pkg/aws/s3/bucket.md

```diff
@@ -0,0 +1,10931 @@
+ {{% /md %}}
+
+            
+            </td>
+        </tr>
+    
+        <tr>
+            <td class="align-top">replication<wbr>Configuration</td>
+            <td class="align-top">
+                
+                <code><a href="#bucketreplicationconfiguration">s3.<wbr>Bucket<wbr>Replication<wbr>Configuration?</a></code>
+            </td>
+            <td class="align-top">{{% md %}} 
+ (Optional)
+A configuration of [replication configuration](http://docs.aws.amazon.com/AmazonS3/latest/dev/crr.html) (documented below).
+ {{% /md %}}
+
+            
+            </td>
+        </tr>
+    
+        <tr>
+            <td class="align-top">request<wbr>Payer</td>
+            <td class="align-top">
+                
+                <code>string?</code>
+            </td>
+            <td class="align-top">{{% md %}} 
+ (Optional)
+Specifies who should bear the cost of Amazon S3 data transfer.
+Can be either `BucketOwner` or `Requester`. By default, the owner of the S3 bucket would incur
+the costs of any data transfer. See [Requester Pays Buckets](http://docs.aws.amazon.com/AmazonS3/latest/dev/RequesterPaysBuckets.html)
+developer guide for more information.
+ {{% /md %}}
+
+            
+            </td>
+        </tr>
+    
+        <tr>
+            <td class="align-top">server<wbr>Side<wbr>Encryption<wbr>Configuration</td>
+            <td class="align-top">
+                
+                <code><a href="#bucketserversideencryptionconfiguration">s3.<wbr>Bucket<wbr>Server<wbr>Side<wbr>Encryption<wbr>Configuration?</a></code>
+            </td>
+            <td class="align-top">{{% md %}} 
+ (Optional)
+A configuration of [server-side encryption configuration](http://docs.aws.amazon.com/AmazonS3/latest/dev/bucket-encryption.html) (documented below)
+ {{% /md %}}
+
+            
+            </td>
+        </tr>
+    
+        <tr>
+            <td class="align-top">tags</td>
+            <td class="align-top">
+                
+                <code>{[key: string]: any}?</code>
+            </td>
+            <td class="align-top">{{% md %}} 
+ (Optional)
+A mapping of tags that identifies subset of objects to which the rule applies.
+The rule applies only to objects having all the tags in its tagset.
+ {{% /md %}}
+
+            
+            </td>
+        </tr>
+    
+        <tr>
+            <td class="align-top">versioning</td>
+            <td class="align-top">
+                
+                <code><a href="#bucketversioning">s3.<wbr>Bucket<wbr>Versioning?</a></code>
+            </td>
+            <td class="align-top">{{% md %}} 
+ (Optional)
+A state of [versioning](https://docs.aws.amazon.com/AmazonS3/latest/dev/Versioning.html) (documented below)
+ {{% /md %}}
+
+            
+            </td>
+        </tr>
+    
+        <tr>
+            <td class="align-top">website</td>
+            <td class="align-top">
+                
+                <code><a href="#bucketwebsite">s3.<wbr>Bucket<wbr>Website?</a></code>
```

Comment:
I think we can do without this, yeah, especially since it's on this page, and the whole page is "in" the `s3` namespace as well, so it seems implicit and okay to omit.